src/main/java/net/snowflake/client/core/SnowflakeMutableProxyRoutePlanner.java

```diff
@@ -0,0 +1,48 @@
+/*
```

Comment:
This class is a wrapper around the SdkProxyRoutePlanner class. It's used so that the nonProxyHosts field can be changed after the HttpClient object has already been built.